modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -9,57 +9,96 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-You can deploy a Mule application to CloudHub from:
-
-* Studio: right-click your project and click *Deploy to Anypoint Platform*, then *Cloud*.
-* Runtime Manager: in the cloud-based version of https://anypoint.mulesoft.com[Anypoint Platform]. Go to the *Runtime Manager* section, then from the *Applications* tab, click *Deploy application*.
-
-You can also deploy to CloudHub through:
+You can deploy Mule applications to CloudHub using:
 
+* Runtime Manager
+* Anypoint Studio
 * The CloudHub API
 * CloudHub CLI
 * Mule-Maven Plugin
 
-== Deploying an Application to CloudHub
+== Naming an Application
+
+When you deploy an application, you must assign a name to it. The application name is the `cloudhub.io` domain name that you use to access your application, therefore, application names must be unique to avoid domain conflicts. 
+
+The application domain identifies your application in Runtime Manager and also provides you with a public URL for accessing the application if it exposes any inbound endpoints. The format of the URL is `+http://myapplicationdomain.cloudhub.io+`. For example, an application named `abcde` is accessible at `+http://abcde.cloudhub.io+`.
+
+When you deploy your application, enter a unique application name. Valid names have a minimum of 3 and a maximum of 42 characters and can contain alphanumeric characters and dashes. Runtime Manager alerts you if the name is available or already reserved by another user.
```

Comment:
Why is the name info repeated on line 22 & here?

Reply:
These feel like task instructions but they are here in a conceptual overview?

Reply:
Not a task. It's conceptual information.